src/openrct2/drawing/scrolling_text.c

```diff
@@ -1566,8 +1566,12 @@ void scrolling_text_set_bitmap_for_ttf(utf8 *text, sint32 scroll, uint8 *bitmap,
             if (scrollPosition > -1) {
                 uint8 *dst = &bitmap[scrollPosition];
 
-                for (sint32 y = 0; y < height; y++) {
-                    if (src[y * pitch + x] != 0) *dst = colour;
+                for (sint32 y = 0; y < height; y++)
+                {
+                    if (src[y * pitch + x] > 92 || (src[y * pitch + x] != 0 && !gConfigFonts.enable_hinting))
```

Comment:
What is `92`?

Reply:
Here, it's an arbitrary threshold for when to interpret a greyscale pixel as a monochrome one for the banners. As my changes change the rendering function to work on the domain [0, 255] rather than [0, 1], a threshold is needed to stick with a monochrome palette for the banners.

I think I fiddled around with it in steps of 8 or 12. It could probably be 90 or 95 as well; we're obviously not bound by RCT2's palette as far as the `src` bitmap is concerned...